app/models/elastic_news_item.rb

```diff
@@ -9,7 +9,7 @@ class ElasticNewsItem
       dynamic: :strict,
       _analyzer: { path: "language" },
       properties: {
-        language: { type: "string", index: :not_analyzed },
+        language: { type: 'keyword', index: true },
         rss_feed_url_id: { type: 'integer' },
```

Comment:
this line of code is already done in https://github.com/GSA/search-gov/pull/256 as well.